Server/src/client_thread.cpp

```diff
@@ -94,12 +97,16 @@ void *ClientThread(void *arg) {
     pthread_cleanup_push(ClientThread_cancel, &cInfo);
 
     try {
+#ifndef REDIS_ENABLED
         // connect to message bus
         cInfo.mbus = new msgBus_kafka(logger, thr->cfg, thr->cfg->c_hash_id);
 
         if (thr->cfg->debug_msgbus)
             cInfo.mbus->enableDebug();
```

Comment:
>enableDebug [](http://example.com/codeflow?start=24&length=11)

You remove debug feature, could you keep it?

Reply:
This debug is used in RdKafka::Conf which is for kafka only, so I don't need it in Redis path.